Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -3319,6 +3319,10 @@ def script_run_polling_command(args: dict, client: CoreClient) -> PollResult:
             response=None,  # since polling defaults to true, no need to deliver response here
             continue_to_poll=True,  # if an error is raised from the api, an exception will be raised
             partial_result=CommandResults(
+                outputs_prefix=f'{args.get("integration_context_brand", "CoreApiModule")}.ScriptRun',
+                outputs_key_field='action_id',
+                outputs=reply,
```

Comment:
will we be returning the same information several times now?

Reply:
Only in the first run. In the rest of the runs we'll have action_id so another flow will run 
https://github.com/demisto/content/pull/35575/files#diff-d973b6667309bcba22ae2bbc58be0bb9b6a44750b343545c4b1654a23c3029deR3299